### PR TITLE
Remove MDN from scaladocs

### LINKED ISSUE #553: Can we get rid of `MDN` in all the ScalaDoc?
It's annoying and useless IMO. Do we really need it? A link to the actual mdn page would be great, but just stating `MDN` seems pretty useless to me.

If it's for licencing issues,

1. we could just make a licencing statement upfront, once, about comments and MDN.
2. descriptions of API aren't copyrightable anyway. Even API itself isn't copyright-able according to Google vs Oracle.

### PR DESCRIPTION
**Pull request by armanbilge**
Fixes #553.

Also it looks like a bunch of other formatting snuck in somehow 🤔

### CODE REVIEW

File: src/main/scala/org/scalajs/dom/Fetch.scala [hunk 1]
```diff
@@ -173,12 +173,12 @@ trait ResponseInit extends js.Object {
 @js.native
 trait Body extends js.Object {
 
-  /** MDN: Contains a Boolean that indicates whether the body has been read.
-    */
+  /*: Contains a Boolean that indicates whether the body has been read.
```
Comment (japgolly) [inline src/main/scala/org/scalajs/dom/Fetch.scala:176]:
```suggestion
  /* Contains a Boolean that indicates whether the body has been read.
```

File: src/main/scala/org/scalajs/dom/Fetch.scala [hunk 2]
```diff
@@ -173,12 +173,12 @@ trait ResponseInit extends js.Object {
...
+   */
   def bodyUsed: Boolean = js.native
 
-  /** MDN: Takes a Response stream and reads it to completion. It returns a promise that resolves with an ArrayBuffer.
-    */
+  /*: Takes a Response stream and reads it to completion. It returns a promise that resolves with an ArrayBuffer.
```
Comment (japgolly) [inline src/main/scala/org/scalajs/dom/Fetch.scala:180]:
```suggestion
  /* Takes a Response stream and reads it to completion. It returns a promise that resolves with an ArrayBuffer.
```

File: src/main/scala/org/scalajs/dom/SVGTypes.scala [hunk 3]
```diff
@@ -2756,38 +2194,30 @@ object SVGPreserveAspectRatio extends js.Object {
   /*
    * Corresponds to value meet for attribute preserveAspectRatio.
    *
-   * MDN
+
```
Comment (japgolly) [inline src/main/scala/org/scalajs/dom/SVGTypes.scala:2197]: woah. Needs another `dom/scalafmt`

File: src/main/scala/org/scalajs/dom/lib.scala [hunk 4]
```diff
@@ -2877,68 +2229,48 @@ abstract class Document extends Node with NodeSelector with DocumentEvent with P
   /** Creates an XPathNSResolver which resolves namespaces with respect to the definitions in scope for a specified
     * node.
     *
-    * MDN.
+    * .
```
Comment (japgolly) [inline src/main/scala/org/scalajs/dom/lib.scala:2232]: remove

**Review by japgolly — CHANGES_REQUESTED** [review]

**Comment by japgolly** [comment]
Awesome work man!

**Comment by armanbilge** [comment]
LOL it was just find replace really. Good catches, let me fix those.

File: src/main/scala/org/scalajs/dom/SVGTypes.scala (hunk 3 above)
Comment (armanbilge) [reply src/main/scala/org/scalajs/dom/SVGTypes.scala:2197]: Ah, it's because the comment starts with `/*` 😡

Comment (japgolly) [reply src/main/scala/org/scalajs/dom/SVGTypes.scala:2194]: lol good catch!

**Review by japgolly — APPROVED** [review]

**Pull request merged**